src/olympia/shelves/models.py

```diff
@@ -27,7 +31,9 @@ class Shelf(ModelBase):
     footer_text = models.CharField(
         max_length=70,
```

Comment:
The tag value isn't stored in the shelf model, so the 70 max length of the field isn't relevant.  Editorial define the tags and define the footer text so it's up to them to take how it may be displayed on the homepage into account.

Reply:
That's what I am saying I believe. This wasn't a problem before because there was no placeholder but now there is `{tag}`.

Reply:
I'm not clear what you're suggesting to change, or how the problem is significantly different to now when l10n can result in strings of an unknown value being displayed for non en-US locales.

Reply:
I am asking questions more than suggesting any change. It's true that we have a potential problem with non en-US locales already.